docs/workflow_solutions/using_anaconda.md

```diff
@@ -282,6 +282,94 @@ Now we can be sure that the correct versions of the software will be installed o
+
+1. Using Google: You may already be familiar with the exact Anaconda package name you require, in the event this is not the case, a simple web engine search with key words would aid your search. For example, a web search for an Anaconda package would be something along the lines of “Anaconda package for `Generic Topic Name`”. Your search results, should return popular package names related to the topic you have searched for. In the sections below, there is an attempt to provide a detailed step-by-step guide on how to find Anaconda packages using “numpy” as an example.
+
+1. Anaconda Cloud: Anaconda Cloud is the primary source for finding Anaconda packages. You can visit [Anaconda Cloud](https://anaconda.org/) and use the search bar to find the package you need. For example, when you get the package name from your web search (using numpy). You will enter name of the package in the search bar as shown below. ![!Landing page of anaconda.org showing search](images/anaconda_search.png)
+
+    Review results of your search, it is advised to use “Artifacts” that are compatible with the platform you are working with, as well as have the most “Favorites” and “Downloads” numbers. Click on the portion that contains the name of the package (highlighted 3 in the image below). 1 highlights the Artifact, Favorite and Downloads numbers, the selection 2 highlights the Channel where this package is stored. ![!Anaconda.org page showing download statistics](images/anaconda_channel_package.png)
+
+    Then follow the installation instructions you see in the image below. ![!Anaconda.org page showing package installation instructions](images/install_anaconda_package.png)
+
+1. Using Conda Search Command: You can use the `conda search <package_name` command directly in your terminal to find packages. Replace `package_name` with the package you would like to search for. To do this on Cheaha, make sure to `module load Anaconda3` first, and follow the instructions to [activate](#activate-an-environment) an environment. Then do `conda search numpy`. You should get a long list of numpy packages. Review this output, but take note of the highlighted portions in the image. The section with a red selection shows the numpy versions that are available, The section with a blue selection shows the channel where each numpy version is stored. ![!Search output from using conda search in Terminal](images/channel_conda_search.png)
+
+    You can then install numpy with a specific version and from a specific channel with.
+
+    ```bash
+    conda install -c conda-forge numpy=2.0.0rc2
+
+    ```
+
+1. Using Specific Channels: You can also get packages using specific Anaconda Channels listed below.
+
+    1. Anaconda Main Channel: The default channel provided by Anaconda, Inc. Visit [Anaconda](https://anaconda.org)
```

Comment:
Hope you don't mind, @iam4tune, I was checking this repo out after you joined us for an _earthaccess_ hack day a couple weeks back!

Are you aware of the new Anaconda terms of service? At minimum I think a warning here is critical for learners to avoid license violations. The Carpentries community is in the process of figuring out how to communicate this to users and what to switch to.

>For sake of clarity, use by government entities and nonprofit entities with over 200 employees or contractors is considered Organizational Use.  Purchasing Starter tier license(s) does not satisfy the Organizational Use paid license requirement set forth in this Section 2.1. Educational Entities will be exempt from the paid license requirement, provided that the use of the Anaconda Offering(s) is solely limited to being used for a curriculum-based course. Anaconda reserves the right to monitor the registration, download, use, installation, access, or enjoyment of the Anaconda Offerings to ensure it is part of a curriculum.  **Utilizing Miniconda to pull package updates from the Anaconda Public Repository without a commercial license (if required by the conditions set forth in Section 2 of this Terms of Service) is considered a violation of the Terms of Service.**

I'd recommend advising against use of the Anaconda or miniconda distributions entirely, and instead direct folks towards [miniforge](https://github.com/conda-forge/miniforge), which offers a 1:1 user experience (including a Miniforge Prompt for Windows users!!) and installs by default with only the conda-forge channel (Anaconda main / "defaults" is intentionally opt-in only) protecting users from license violations!

Reply:
Hello @mfisher87 thanks for the information. I have shared this with my manager, and we would discuss next steps.

Reply:
Heck yeah! I'll be honest, I'm a bit tilted by this license stuff, but it feels like a familiar story at this point :( So thankful for the Conda Forge community! :pray:

Reply:
Hi @mfisher87, thanks for letting us know about this. We were not aware and will have to consider how best to pivot.

Reply:
For posterity, I'm going to approve and merge this request for now, with an eye toward making a transition at a future date once the management team decides the best course of action. I've already got a proposal for this but it's a big lift. In the meantime I'd rather not unfairly disrupt our researchers' work.

Reply:
>I've already got a proposal for this but it's a big lift

Just curious, did you rule out the miniforge approach? I see that as the lightest-weight way to transition folks off the proprietary ecosystem. Their Windows support is even on point!